bower.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "collabosphere",
+  "description": "Collabosphere Instructure Canvas LTI Tools",
+  "version": "0.1.0",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/ets-berkeley-edu/collabosphere.git"
+  },
+  "dependencies": {
+    "angular": "latest",
```

Comment:
I assume this stuff will be used for the UI counterpart to this PR?

Can you add some instructions to the README that explain how to pull in these assets through bower?